src/pages/workspace/taxes/WorkspaceTaxesPage.tsx

```diff
@@ -235,7 +235,7 @@ function WorkspaceTaxesPage({
             onPress={() => {}}
             options={dropdownMenuOptions}
             buttonSize={CONST.DROPDOWN_BUTTON_SIZE.MEDIUM}
-            customText={translate('workspace.common.selected', {selectedNumber: selectedTaxesIDs.length})}
+            customText={translate('workspace.common.selected', selectedTaxesIDs.length)}
```

Comment:
@brandonhenry  This change will crash the app. Can you please check it on your side? I have tested it, and it crashes for me.

Reply:
https://github.com/Expensify/App/assets/35371050/5f00a1fb-cf6d-48f7-92ca-44cc0dc0e104